dev/com.ibm.ws.jaxrs.2.1_fat_extended/build.gradle

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
```

Comment:
2018 - though only update this if you make other changes.

Reply:
You are right, I forgot that our build.gradle file was previously removed for being empty and I had to re add it with this change.